pandas/tools/tests/test_pivot.py

```diff
@@ -402,6 +426,24 @@ def test_margins_no_values_two_row_two_cols(self):
         result = self.data[['A', 'B', 'C', 'D']].pivot_table(index=['A', 'B'], columns=['C', 'D'], aggfunc=len, margins=True)
         self.assertEqual(result.All.tolist(), [3.0, 1.0, 4.0, 3.0, 11.0])
 
+    def test_pivot_table_with_margins_set_margin_name(self):
+        for margin_name in ['foo', 'one']:
```

Comment:
add the issue name here. Try a name which is invalid (e.g. we are going to require it be a string), IOW a tuple would be bogus.